config/seata-config-zk/src/main/java/io/seata/config/zk/ZookeeperConfiguration.java

```diff
@@ -247,4 +252,23 @@ public void handleDataDeleted(String s) {
         }
     }
 
+    private ZkSerializer getZkSerializer() {
+        ZkSerializer zkSerializer = null;
+        String serializer = FILE_CONFIG.getConfig(FILE_CONFIG_KEY_PREFIX + SERIALIZER_KEY);
+        if (StringUtils.isNotBlank(serializer)) {
+            try {
+                Class<?> aClass = Class.forName(serializer);
+                zkSerializer = (ZkSerializer) aClass.newInstance();
+            } catch (ClassNotFoundException cfe) {
+                LOGGER.error("No zk serializer class found, serializer:{}", serializer, cfe);
+            } catch (Throwable cause) {
+                LOGGER.error("found zk serializer encountered an unknown exception", cause);
+            }
+        }
+        if (zkSerializer == null) {
+            zkSerializer = new DefaultZkSerializer();
```

Comment:
How about use log to tell users, zkSerializer would use DefaultZkSerializer

Reply:
Done.